src/components/duelling-picklist/picklist-group/picklist-group.spec.tsx

```diff
@@ -321,10 +314,38 @@ describe("PicklistGroup component", () => {
     });
   });
 
+  describe("children", () => {
+    it("does not throw when incorrect children is provided", () => {
```

Comment:
```suggestion
    it("does not throw when incorrect children are provided", () => {
```

just improving English grammar :-)